lib/Core/SQLiteBuildDB.cpp

```diff
@@ -23,6 +23,7 @@
 #include <cerrno>
 #include <cstring>
 #include <mutex>
+#include <unordered_map>
```

Comment:
Can we use `llvm:DenseMap` rather than an `std` data structure here? The LLVM ones are generally faster.